hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module main
```

```suggestion
module github.com/diletech/hw-test/hw01_hello_otus
```
В названии модуля следует указать полный путь до репозитория, чтобы он был доступен для загрузки и импорта